### PR TITLE
code snippet AWSO fix

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/quickstart.md [hunk 1]
```diff
@@ -5,21 +5,48 @@ sidebar_label: AWS Observability Quickstart
...
+
+```
+.\DeployAWSOWin.ps1 <SUMO_ACCESS_ID> <SUMO_ACCESS_KEY>
+```
+
+
+</TabItem>
+<TabItem value="Linux / Mac">
+
+For :
```
Comment (sumoanema) [inline docs/observability/aws/quickstart.md:42]: This seems unnecessary. I guess you already defined tabs for linux vs windows. This "For" was in the doc to point to the same thing.

Comment (kimsauce) [reply docs/observability/aws/quickstart.md:42]: great catch!

File: docs/observability/aws/quickstart.md [hunk 2]
```diff
@@ -5,21 +5,48 @@ sidebar_label: AWS Observability Quickstart
...
+Copy the file available here. Save it as “DeployAWSOPosix.sh”. Run the command:
```
Comment (sumoanema) [inline docs/observability/aws/quickstart.md:43]: here - should have an embedded link which I had pasted in my suggested doc. Its missed here. Here is the link - https://github.com/SumoLogic/sumologic-solution-templates/blob/master/aws-observability/scripts/AWSOAutoSetupScript/DeployAWSOPosix.sh

File: docs/observability/aws/quickstart.md [hunk 3]
```diff
@@ -5,21 +5,48 @@ sidebar_label: AWS Observability Quickstart
...
+className="unique-tabs"
+defaultValue="Windows"
+values={[
+{label: 'Windows', value: 'Windows'},
+{label: 'Linux / Mac', value: 'Linux / Mac'}
+]}>
+
+<TabItem value="Windows">
+
+Copy the file available here. Save it as “DeployAWSOWin.ps1”. Run the command:
```
Comment (sumoanema) [inline docs/observability/aws/quickstart.md:32]: Here- should have an embedded link - https://github.com/SumoLogic/sumologic-solution-templates/blob/master/aws-observability/scripts/AWSOAutoSetupScript/DeployAWSOWin.ps1

**Pull request merged**